skribenten-backend/build.gradle.kts

```diff
@@ -71,9 +72,12 @@ dependencies {
     implementation("org.jetbrains.exposed:exposed-core:$exposedVersion")
     implementation("org.jetbrains.exposed:exposed-dao:$exposedVersion")
     implementation("org.jetbrains.exposed:exposed-jdbc:$exposedVersion")
+    implementation("org.jetbrains.exposed:exposed-json:$exposedVersion")
     implementation("org.postgresql:postgresql:42.7.2")
     implementation("com.zaxxer:HikariCP:5.1.0")
 
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:$exposedVersion")
```

Comment:
Fungerer exposed-json kun med kotlinx-serialization? Tenkte bare det ble litt kjøtt på flesk med både jackson og kotlinx-serialization for json.